recipes/dudes/meta.yaml

```diff
@@ -3,12 +3,12 @@ package:
   version: "0.07"
 
 source:
-  fn: dudes_v0_07.tar.gz
-  url: https://sourceforge.net/projects/dudes/files/dudes_v0_07.tar.gz
-  md5: 39b755dd132f7377d3c00cb65977fc90
+  fn: v0.07.tar.gz 
```

Comment:
It is recommended to keep it as `dudes_0.07.tar.gz`i.e. `name_version.tar.gz` to avoid name collisions in the cache.

Reply:
Fixed! Tks.